.github/CONTRIBUTING.md

```diff
@@ -18,8 +18,9 @@ If you'd like to write some code for nf-core/eager, the standard workflow is as
 1. Check that there isn't already an issue about your idea in the [nf-core/eager issues](https://github.com/nf-core/eager/issues) to avoid duplicating work
     * If there isn't one already, please create one so that others know you're working on this
 2. [Fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) the [nf-core/eager repository](https://github.com/nf-core/eager) to your GitHub account
-3. Make the necessary changes / additions within your forked repository
-4. Submit a Pull Request against the `dev` branch and wait for the code to be reviewed and merged
+3. Make the necessary changes / additions within your forked repository (following [code contribution guidelines](https://github.com/nf-core/eager/blob/dev/docs/code_contribution.md))
```

Comment:
should this link be more general instead of on the `dev` branch?

Reply:
As it's in contributing code section, it should specifically go onto `dev`!